lightning/src/routing/router.rs

```diff
@@ -495,6 +493,13 @@ impl<'a> PaymentPath<'a> {
 		self.hops.last().unwrap().0.fee_msat
 	}
 
+	fn get_path_penalty_msat(&self) -> u64 {
+		if self.hops.len() < 1 {
+			return u64::max_value();
+		}
+		self.hops.first().unwrap().0.path_penalty_msat
```

Comment:
Just to double check: with references to [this comment](https://github.com/lightningdevkit/rust-lightning/blob/main/lightning/src/routing/router.rs#L327) I'm assuming here that the _first_, not the _last_ hop in the `PaymentPath` features the overall penalty.

Reply:
Yea, its built from the `ordered_hops` vec in the `'path_walk` loop, which swaps the order from the graph walk by walking backwards.

Reply:
Yes, saw that, just wanted to be sure I'm not missing something. Thank you!